pkg/libcose/doc.txt

```diff
@@ -0,0 +1,39 @@
+/**
+ * @defgroup pkg_libcose  libcose for RIOT
+ * @ingroup  pkg
+ * @brief    Constrained node COSE library
+ * @see      https://github.com/bergzand/libcose
+ *
+ * Usage
+ * -----
+ *
+ * Add as a package in the Makefile of your application:
+ *
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ {.mk}
+ * USEPKG += libcose
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+  * The use of the library itself [is described in the libcose documentation](https://bergzand.github.io/libcose/).
```

Comment:
Maybe also add a reference to the RIOT test here for a small usage example?

Reply:
Added, along with a reference to SUIT. (The idea is kind of to guide novice users towards existing high-level abstractions before they see "hey cool crypto I'll try it out" and re-invent the wheel.)